integration-tests/testconfig/ccip/ccip.toml

```diff
@@ -242,16 +242,12 @@ addresses_to_fund = [
 ephemeral_addresses_number = 0
 
 [Load.CCIP.Load]
-# replace this with the loki endpoint of the crib stack
-LokiEndpoint = "http://loki.localhost/loki/api/v1/push"
 # MessageTypeWeights corresponds with [data only, token only, message with data]
 MessageTypeWeights = [100,0,0]
 # each destination chain will receive 1 incoming request per RequestFrequency for the duration of LoadDuration
-RequestFrequency = "10s"
-LoadDuration = "1m"
-# test duration
-TestDuration = "5h"
-# destination chain selectors to send messages to
-EnabledDestionationChains = [3379446385462418246]
+RequestFrequency = "30s"
+LoadDuration = "1h"
+# number of destination chains
+NumDestinationChains = 30
 # Directory where we receive environment configuration from crib
 CribEnvDirectory = "/Users/austin.wang/ccip-core/repos/crib/deployments/ccip-v2/.tmp"
```

Comment:
I know it was there before, but are you sure about keeping hardcoded paths to local env here?